opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -105,17 +239,34 @@ namespace Opm {
                 throw std::runtime_error(std::string("Parser::openFile fails: ") + fs_error.what());
             }
 
-            std::ifstream *ifs = new std::ifstream(inputFileCanonical.string().c_str());
+            const auto closer = []( std::FILE* f ) { std::fclose( f ); };
+            std::unique_ptr< std::FILE, decltype( closer ) > ufp(
+                    std::fopen( inputFileCanonical.string().c_str(), "rb" ),
+                    closer
+                );
 
             // make sure the file we'd like to parse is readable
-            if (!ifs->is_open()) {
+            if (!ufp ) {
                 throw std::runtime_error(std::string("Input file '") +
                                          inputFileCanonical.string() +
                                          std::string("' is not readable"));
             }
 
-            inputstream.reset( ifs );
-            dataFile = inputFileCanonical;
+            /*
+             * read the input file C-style. This is done for performance
+             * reasons, as streams are slow
+             */
+            auto* fp = ufp.get();
+            std::string buffer;
+            std::fseek( fp, 0, SEEK_END );
+            buffer.resize( std::ftell( fp ) );
+            std::rewind( fp );
+            std::fread( &buffer[ 0 ], 1, buffer.size(), fp );
```

Comment:
No, because it is not non-const until C++17: http://en.cppreference.com/w/cpp/string/basic_string/data

I **really** wish I could because this is sort-of hacky, but it's the only way (to my knowledge) to make it work.